eggbox_potential_sampler/sampling_data_view/tests/test_sampling_data_view.py

```diff
@@ -65,11 +64,11 @@ def add_data_points(self):
         for datum in [
                 (1, 3, 8), (2, 5, 7.7), (3, 2.6, 7.6), (4, 2.5, 7.62),
                 (5, 2.47, 7.543), (6, 2.465, 7.54)]:
-            self.analysis_model.add_evaluation_step(datum)
+            self.analysis_model._add_evaluation_step(datum)
```

Comment:
Same here. Could you please check if `self.analysis_model.notify(datum)` does the job?